build-scripts/gulp/compress.js

```diff
@@ -60,9 +66,19 @@ const compressHassioZopfli = () =>
     paths.hassio_output_latest,
     true
   );
+const compressLandingPageZopfli = () =>
+  compressDistZopfli(
+    paths.landingPage_output_root,
+    paths.landingPage_output_latest,
+    true
+  );
 
 gulp.task("compress-app", gulp.parallel(compressAppBrotli, compressAppZopfli));
 gulp.task(
   "compress-hassio",
   gulp.parallel(compressHassioBrotli, compressHassioZopfli)
 );
+gulp.task(
+  "compress-landing-page",
+  gulp.parallel(compressLandingPageBrotli, compressLandingPageZopfli)
```

Comment:
Does the go web server we use support this?

Reply:
:shrug: Just did the same as hassio build, I will test it out.

Reply:
No it doesn't support this, just with 3rd party libs. I will remove it from the build.